scipy/stats/_rvs_sampling.py

```diff
@@ -60,17 +59,24 @@ def rvs_ratio_uniforms(pdf, umax, vmin, vmax, size=1, c=0, random_state=None):
     `V/U + c` if `(U, V)` are also in `A` which can be directly
     verified.
 
+    The algorithm is not changed if one replaces `pdf` by k * `pdf` for any
```

Comment:
should the second pdf on this line instead read `k * f(x)`? and state that `f(x)` is proportional to a density? 

I've heard `f(x)` in this setting called a ***kernel*** (Bayesian statistics) but not sure how standard that nomenclature is more generally.

Reply:
the input is pdf, so I thought k * pdf would be ok

Reply:
ah, I see, ok. Given that we're now making this explicit in the docstring, is there a need for a test that exhibits (and assures) the same behavior with `pdf` vs `k * pdf` as input? Or is there already a test of that?

You could keep the old doctest (with normal distribution), and include the new doctest with the lambda function, to illustrate to the user, what do you think?

Reply:
hm, I think it might be even a bit too much since it is really just a scaling with 1/sqrt(2 Pi) and it is common to drop the constants. I prefer to keep it as it is.